lighthouse-cli/cli-flags.ts

```diff
@@ -17,8 +17,10 @@ export interface Flags {
       view: boolean, maxWaitForLoad: number
 }
 
-export function getFlags() {
-  return yargs.help('help')
+export function getFlags(fakeInput?: string) {
```

Comment:
nice, but can we call the parameter something else? It seems like a legitimate use, not just for testing, so what about `manualInput`, `manualArgv`, `argvString`, or something something :)